perun-core/src/main/java/cz/metacentrum/perun/core/impl/Auditer.java

```diff
@@ -171,13 +169,7 @@ public Auditer() {
 
 	public void setPerunPool(DataSource perunPool) throws InternalErrorException {
 		this.jdbc = new JdbcPerunTemplate(perunPool);
-		if(Compatibility.isOracle()) {
-			OracleLobHandler oracleLobHandler = new OracleLobHandler();
-			oracleLobHandler.setNativeJdbcExtractor(new CommonsDbcpNativeJdbcExtractor());
-			lobHandler = oracleLobHandler;
-		} else {
-			lobHandler = new DefaultLobHandler();
-		}
+		this.lobHandler = new DefaultLobHandler();
```

Comment:
I'm not even sure, we need LobHandler at all, since we only store (N)CLOB (for Oracle) and it can be read as ``.getString()`` by JDBC. But it probably shouldn't be part of this change. Just suggestions to test and implement.

Reply:
I do not know, I have just removed the branch for Oracle because ti was complaining that default lob handler should be used since some version of Oracle JDBC driver.